tests/FSharp.Core.UnitTests/FSharp.Core/CreateComparersRegression.fsx

```diff
@@ -3,4 +3,4 @@
 
 #r @"..\..\..\..\Release\net40\bin\FSharp.Core.UnitTests.dll"
 
-FSharp.Core.UnitTests.FSharp_Core.Microsoft_FSharp_Core.ComparersRegression.createData ()
```

Comment:
I shortned a lot of names since they are crazy long in the Test Explorer and there was redundancy in the names in any case